docker/image_types.yaml

```diff
@@ -49,7 +49,6 @@
     - 'parent_image=gcr.io/fuzzbench/builders/coverage/{benchmark}-intermediate'
   depends_on:
     - 'coverage-{benchmark}-builder-intermediate'
-    - 'base-image'
```

Comment:
What's the reason for this removal?

Reply:
> What's the reason for this removal?

I do not know, it comes with this branch of yours : )

Reply:
Would you prefer me to revert this?

Reply:
I think this is causing the CI failure as we discussed